apple-nav/src/components/NavItems.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import NavItem from "./NavItem"
+
+const NavItems=(props)=> {
+    return (
+      <div style={{display:"flex",justifyContent:"space-around"}} >
+       {props.myData.map((item,i)=>(
+           <NavItem key={item.navName} item={item}/>
+       ))}
```

Comment:
You could also create separate CSS files or utilize styled components instead of placing styles inline directly into the `style` attribute.